src/app/server/CommissioningWindowManager.h

```diff
@@ -59,8 +59,13 @@ class CommissioningWindowManager : public Messaging::UnsolicitedMessageHandler,
 
     static constexpr System::Clock::Seconds16 MaxCommissioningTimeout()
     {
+#if CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+        // per new spec change/extension - Extended Announcement Duration up to 48h
```

Comment:
```suggestion
        // per spec - Extended Announcement Duration up to 48h
```

Reply:
@mkardous-silabs  I got the actual proper info from the new spec now: 
`// Specification section 2.3.1 - Extended Announcement Duration up to 48h`